ixwebsocket/IXSocketOpenSSL.cpp

```diff
@@ -283,17 +327,19 @@ namespace ix
             }
             SSL_set_fd(_ssl_connection, _sockfd);
 
-            // SNI support
-            SSL_set_tlsext_host_name(_ssl_connection, host.c_str());
+            if (!_tlsOptions.isPeerVerifyDisabled()) {
```

Comment:
I said that somewhere else but curly braces should be on their own line.

Reply:
fixed